components/Heart.tsx

```diff
@@ -0,0 +1,37 @@
+import { Icon } from "@equinor/eds-core-react";
+import React, { useState } from "react";
+import { favorite_outlined, favorite_filled } from "@equinor/eds-icons";
+
+export default function Heart(props: {
+  isFavourite: boolean;
+  fave: () => void;
+  unfave: () => void;
+}): JSX.Element {
+  const [isHighlighted, setIsHighlighted] = useState(false);
+
+  return (
+    <div
+      style={{ alignSelf: "center", margin: "12px" }}
+      onMouseEnter={(e) => {
+        e.stopPropagation();
+        setIsHighlighted(true);
+      }}
+      onMouseLeave={(e) => {
+        e.stopPropagation();
+        setIsHighlighted(false);
+      }}
```

Comment:
Nicer to do this in CSS if you can.

Reply:
Mmm I've just tried but here we are rendering a different icon depending on whether the mouse is over the heart or not. If the mouse is elsewhere we render <Icon data={heart_outlined}> vs <Icon data={heart_filled}> when hovering. I'm not sure that can be coded in CSS?

Reply:
🙆‍♂️